UTILS/parse-async-WorkflowConfig.py

```diff
@@ -193,8 +193,9 @@ def parse_important_DPL_args(cmds, flat_config):
          s2 = extract_args(tokens, '--corrmap-lumi-mean')
```

Comment:
Hello @sawenzel , @shahor02 , 
Do we want to wait for this, or merge? The PRs in O2 are merged, so this in principle could go in.
Cheers,
Chiara

Reply:
Since we don't currently use distortions in the MC, I am not sure what was the point of copying these options from the ITS-TPC matching.
For me it is fine to merge this, but then, it obviously cannot be used on the async-stable w/o porting PR#12205 and PR#12237.

Reply:
Hello @sawenzel , 
Do you agree to merge?
@shahor02 , yes, the dependence to O2 is clear.
Chiara